### PR TITLE
Add posibility to use absolute file path in outputReport.filePath

### PR DESCRIPTION
**Pull request by ars-java**
Issue: https://github.com/webpack-contrib/eslint-loader/issues/265

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -96,18 +98,34 @@ function printLinterOutput(res, config, webpack) {
...
-              .map(function(r) {
-                return r.source;
-              })
-              .join("\n")
-          }
-        );
-        webpack.emitFile(filePath, reportOutput);
+
+        if (path.isAbsolute(config.outputReport.filePath)) {
+          fs.writeFile(config.outputReport.filePath, reportOutput, function(
```
Comment (evilebottnawi) [inline index.js]: Bad idea use `fs` inside loader, it is break code when you in development mode (we use `memory` fs )

Comment (ars-java) [reply index.js]: @evilebottnawi What do you think about adding outputReport.stream config option to write into custom stream? We can open, for example, file stream in config and write report content into it.

Comment (evilebottnawi) [reply index.js]: What is use case to use absolute?

Comment (ars-java) [reply index.js]: To save report into another folder than `output.path`. I think there's no need to restrict it. In my project I have `reports` folder and I want to save all reports into it. There's no way to do it now.

Comment (evilebottnawi) [reply index.js]: You can use `relative` paths to you folder

Comment (nooks) [reply index.js]: o_O

Comment (ars-java) [reply index.js]: Why do you think that it's better than to use custom pluggable stream?

Comment (evilebottnawi) [reply index.js]: It is not better, just ask the questions. Anyway we should avoid using `fs` inside loader, so this PR need to rework, feel free to implement other approach

Comment (ricardogobbosouza) [reply index.js]: How can we do this @evilebottnawi?

Comment (evilebottnawi) [reply index.js]: @ricardogobbosouza hm, look here one solution, use read `fs`, maybe we should add support `fs` in loaders

**Comment by evilebottnawi** [comment]
Why we need use absolute path here? Somebody can provide use case?

File: index.js (hunk 1 above)
Comment (ricardogobbosouza) [reply index.js]: Can we use `fs`?

Comment (evilebottnawi) [reply index.js]: yes, looks no other way to solve problem

**Pull request closed without merging**